### DISCUSSION TITLE
node red home assistant websocket Version 0.62.1 requires Node-RED 3.1.1+

### DISCUSSION THREAD

**Discussion by koenroels2** [hassio-addons/addon-node-red#1775] [Q&A]
Hi, 
Its impossible to install the home assistant websocket 0.62.1 with the actual node-red 3.1.0.
Do you think it's possible to have an update with the 3.1.1+ version or higher?
also problems with HACS installation with HA 2023.12... Not possible to intall node-red compagnon.
thanks
Koen

**Comment by LucasJanin** [comment]
Same issue on my side. 
I intend to install Node-RED outside of Hassio.

**Comment by blobyNob** [comment]
Hi, I have this problem as well. 0.62.1 prevents node red addon functioning once installed. I now cannot run any flows.

**Comment by koenroels2** [comment]
Hi,
only solution  is to install the las version. then: restart and the websocket disappear. re-install the websocket. Then IMPORTANT:never restart de NODE-RED. only restart Home assistant if necessary. 
If I must restart the Node red , you need to install again the last websocket but you see that the version 0.59.0 is present.
I wait after the update of the NODE-RED Current version: 16.0.2 .
We must wait after the Node-red 3.1.1+ minimum.
WAIT AND SEE LATER.
